GameProject/Engine/Components/Explosion.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Explosion.h"
+
+#include "Engine/Entity/Entity.h"
+#include <cstdlib>
+#include <ctime>
+
+Explosion::Explosion(Entity * host, unsigned explosionDebris)
+	: Component(host, "Explosion")
+{
+	ParticleManager& pm = ParticleManager::get();
+
+	glm::vec3 entityPos = this->host->getTransform()->getPosition();
+
+	srand(static_cast <unsigned> (time(0)));
+	float LO = -1.0;
```

Comment:
```suggestion
	float LO = -1.0f;
```

Same at lines 16, 18, 19, 25, 26, 29, 31, 49 and 51